proxy/src/graphql/schema.rs

```diff
@@ -260,24 +260,55 @@ enum ObjectType {
     Blob,
 }
 
+/// Contextual information for an org registration message.
+#[derive(juniper::GraphQLObject)]
+struct OrgRegistration {
+    /// The ID of the org.
+    org_id: String,
+}
+
+/// Contextual information for an org unregistration message.
+#[derive(juniper::GraphQLObject)]
+struct OrgUnregistration {
+    /// The ID of the org.
+    org_id: String,
+}
+
 /// Contextual information for a project registration message.
 #[derive(juniper::GraphQLObject)]
 struct ProjectRegistration {
-    /// Domain namespace.
-    domain: String,
-    /// Actual project name, unique under domain.
-    name: String,
+    /// Actual project name, unique under org.
+    project_name: String,
+    /// The org under which to register the project.
+    org_id: String,
 }
 
 /// Message types supproted in transactions.
 enum Message {
+    /// Registration of a new org.
```

Comment:
All the additions here regarding messages seem to be leftovers. Can we remove them.

Reply:
@xla by removing these, `fn messages` (line 321) can't be total since it cannot convert all `registry::Message` to their `schema.rs` counterpart, requiring changing the code for that. It appears more sensible to me to thus leave these new message types in.

Reply:
Right, that makes all the sense.